Control/public/pages/CalibrationRuns/CalibrationRuns.model.js

```diff
@@ -59,6 +59,26 @@ export class CalibrationRunsModel extends Observable {
     this.notify();
   }
 
+
+  /**
+   * Request a reset of the calibration page including the calibration configurations and reload
+   * the calibration runs if the reset was successful
+   * @return {void}
+   */
+  async requestReload() {
```

Comment:
For me it feels weird to name it `requestReload` I would directly name it `reload`, `refresh`

Reply:
I thought about that but then I used `requestReload` as:
- not any user has access to reload, and this is a request verified by the server and only if successful it will start the process.
- reload/refresh I thought it might incline the developer thinking it is a refresh or reload of the page.

Reply:
Yes I prefer `refreshCalibrationConfigurations`

> not any user has access to reload, and this is a request verified by the server and only if successful it will start the process.
In that case the function should return and the button should not be available no?

Reply:
Renamed